install/kubernetes/helm/istio/charts/pilot/templates/deployment.yaml

```diff
@@ -67,49 +67,6 @@ spec:
           - name: istio-certs
             mountPath: /etc/certs
             readOnly: true
-        - name: istio-proxy
```

Comment:
For upgrade from 0.8 (with alpha3) - you must also change the default port for grpc/mtls from 15012 to 15011.